terraform/environments/core-logging/route53_resolver_logs.tf

```diff
@@ -1,3 +1,8 @@
+provider "aws" {
```

Comment:
if needed, this should go to providers.tf

this change on its own will not have the desired effect. Note, you will need to reference it in your resources for it to have an effect

Reply:
Made amendments and pushed back up